apps/freeablo/faworld/gamelevel.cpp

```diff
@@ -86,11 +86,11 @@ namespace FAWorld
 
     void GameLevel::update(bool noclip)
     {
-        for (size_t i = 0; i < mActors.size(); i++)
-        {
-            Actor* actor = mActors[i];
+        for (auto& actor : mActors)
+            actor->recalculateStats();
+
+        for (auto& actor : mActors)
```

Comment:
No, because a damage calculation later on might depend on the stats of another actor whose stats haven't been refreshed yet. Eg: I damage you, I've been refreshed, you haven't, so maybe your armor value is incorrect because you just changed your gear in the last tick. It would be only one frame off, but might as well be correct and avoid surprises.